apps/test/unit/applab/ExporterTest.js

```diff
@@ -674,6 +684,22 @@ describe('The Exporter,', function () {
       );
     });
 
+    it("should allow you to use startWebRequest without the XHR proxy", (done) => {
+      runExportedApp(
+        `var webRequestPromise = new Promise(function (resolve, reject) {
```

Comment:
This is clever, by the way - good way to test async code.  A little misleading since we don't support Promises in the interpreter, but works great here.

Reply:
yes, fortunately, it is a test of "exported" code, which doesn't run in the interpreter and can take full advantage of things like promises.